assets/javascripts/initializers/ai-image-caption.js

```diff
@@ -46,6 +53,18 @@ export default apiInitializer("1.25.0", (api) => {
             imageCaptionPopup.imageSrc = imageSrc;
             imageCaptionPopup.imageIndex = imageIndex;
             imageCaptionPopup.newCaption = caption;
+
+            if (site.mobileView) {
+              // Auto-saves caption on mobile view
+              const matchingPlaceholder =
+                composer.model.reply.match(IMAGE_MARKDOWN_REGEX);
+              const match = matchingPlaceholder[imageIndex];
+              const replacement = match.replace(
+                IMAGE_MARKDOWN_REGEX,
+                `![${imageCaptionPopup.newCaption}|$2$3$4]($5)`
+              );
+              appEvents.trigger("composer:replace-text", match, replacement);
```

Comment:
@keegangeorge How difficult would it be to add a happy-path system test for this? 

- on mobile
- open composer
- upload image
- press AI caption button
- check that markup has an AI-generated caption